presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -424,7 +431,7 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateTask(
     const protocol::OutputBuffers& outputBuffers,
     std::shared_ptr<velox::core::QueryCtx> queryCtx,
     long startProcessCpuTime) {
-  std::shared_ptr<exec::Task> execTask;
+  std::shared_ptr<exec::Task> veloxTask;
```

Comment:
Nice rename, but it would be better to do this in a separate PR. Mixing trivial refactoring with meaningful changes makes it hard to review the code.

Reply:
Reverted.